CHANGELOG.txt

```diff
@@ -2,6 +2,8 @@ master
 ======
 
 Changes:
+ - obspy.clients.fdsn
+   * Natural Resources Canada (NRCAN) added to list of known clients
```

Comment:
can you move this down a bit? we like to keep `core` at the top and then do alphabetical from there =)

Reply:
Yes, done. Sorry for the delay, I need to review my notification settings.